libs/custom-paper-handler/src/driver/driver.test.ts

```diff
@@ -0,0 +1,587 @@
+import { assert } from '@votingworks/basics';
+import { mocks } from '@votingworks/custom-scanner';
+import { Buffer } from 'buffer';
+import { findByIds, WebUSBDevice } from 'usb';
+import { Uint16 } from '@votingworks/message-coder';
+import {
+  GENERIC_ENDPOINT_OUT,
+  REAL_TIME_ENDPOINT_IN,
+  REAL_TIME_ENDPOINT_OUT,
+  getPaperHandlerWebDevice,
+  PaperHandlerDriver,
+  RealTimeRequestIds,
+  ReturnCodes,
+  GENERIC_ENDPOINT_IN,
+} from './driver';
+import { TOKEN, NULL_CODE } from './constants';
+import { PrinterStatus, ScannerStatus } from './sensors';
+import { setUpMockWebUsbDevice } from './test_utils';
+
+/**
+ * message-coder (new) encodes as Buffers, but the prototype driver (legacy) sends messages as Uint8Array.
+ * This means the type of input for calls to webDevice.transferOut will differ between the legacy and new implementations.
+ * We wrap test expectations in formatTransferOutArg now so we can easily change them in the future.
```

Comment:
This function was originally for interop between old and new driver, but with the way the PRs are structured now, we can probably remove this function in the following PR